### PR TITLE
[PHP 8.3] `opcache.preload_user` is not required when run as root

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Part of #2796

### CODE REVIEW

**Review by arnaud-lb — APPROVED** [review]
Thank you!

File: reference/opcache/ini.xml [hunk 1]
```diff
@@ -962,6 +962,13 @@
       startup as root before switching to an unprivileged system user. Preloading as root is not
       allowed by default for security reasons, unless this directive is explicitly set to
       <literal>root</literal>.
+
+      <note>
+       <simpara>
+        As of PHP 8.3.0 and when using the CLI or phpdbg SAPIs, the directive isn't required
+        by the preloading to be set when running as root.
```
Comment (jimwins) [inline reference/opcache/ini.xml:969]:
```suggestion
        As of PHP 8.3.0, this directive does not need to be set in order to allow preloading while
        running as root when using the &cli.sapi; or <link linkend="book.phpdb">phpdbg SAPI</link>.
```

Maybe we don't even need to bump this out as a `<note>` but just add the sentence to the preceding paragraph.

**Review by jimwins — CHANGES_REQUESTED** [review]

File: reference/opcache/ini.xml [hunk 2]
```diff
@@ -961,7 +961,9 @@
       Lets the preloading to be run as the specified system user. This is useful for servers that
       startup as root before switching to an unprivileged system user. Preloading as root is not
       allowed by default for security reasons, unless this directive is explicitly set to
-      <literal>root</literal>.
+      <literal>root</literal>. As of PHP 8.3.0, this directive does not need to be set in order
+      to allow preloading while running as root when using the &cli.sapi; or
+      <link linkend="book.phpdb">phpdbg SAPI</link>.
```
Comment (jimwins) [inline reference/opcache/ini.xml:966]: You should start the sentence on a line of its own. (It's a convention called "[semantic line breaks](https://sembr.org/)" that should help with keeping down formatting churn for translations.)

LGTM with that change.

**Review by jimwins — APPROVED** [review]

File: reference/opcache/ini.xml (hunk 2 above)
Comment (alexandre-daubois) [reply reference/opcache/ini.xml]: I wasn't aware this had a name. Thank you Jim, it's updated!

**Pull request merged**